pkg/services/ngalert/notifier/external_alertmanager.go

```diff
@@ -121,30 +123,62 @@ func (am *externalAlertmanager) ListSilences(ctx context.Context, filter []strin
 	return res.Payload, nil
 }
 
-func (am *externalAlertmanager) GetStatus() apimodels.GettableStatus {
-	return apimodels.GettableStatus{}
+func (am *externalAlertmanager) GetAlerts(ctx context.Context, active, silenced, inhibited bool, filter []string, receiver string) (apimodels.GettableAlerts, error) {
+	params := amalert.NewGetAlertsParamsWithContext(ctx).
```

Comment:
I don't love that context is put in params instead of passed to `GetAlerts`. It's just no idiomatic Go, so as a reader of the code I feel I need to spend time to understand if there's a point of doing it this way instead of the idiomatic way. Also, is there a reason why it's done this way? 😄